src/System.Threading.Tasks/tests/Task/TaskStatusTest.cs

```diff
@@ -159,7 +159,7 @@ internal void RealRun()
                         // without Detached options and current status of the child isn't RanToCompletion or Faulted yet
                         //
 
-                        Task.Delay(100).Wait();
+                        Task.Delay(1).Wait();
```

Comment:
The main difference is that the SpinWait will burn the CPU, whereas this will put the thread to sleep.  In the other case I switched to a small amount of spinning because the goal of that code as I understood it was to do a tiny amount of work in the body, so spinning fit better.

In general, though, there appear to be lots of things in these tests that could stand to be rewritten.  It's not clear what many of the tests are trying to verify nor that they're successfully doing so.